community-meetings/2022-01-07.md

```diff
@@ -0,0 +1,18 @@
+# Agenda for the GoReleaser Community Meeting - January 7, 2022
+
+## Links for the participants
```

Comment:
Is there a Zoom/Meet/something link for people who want to join and talk?

Reply:
We thought about doing it via Streamyard on youtube. I need to grab @caarlos0 to create the preview on the goreleaser youtube channel.

Reply:
Wouldn't this make it harder to collaborate (text Vs speech)? 🤔

Reply:
I think if we want to go more collab, Zoom would be good as we can stream to youtube too.

Streamyard has a better L&F, the core contributer can be directly in the session and via twitch & youtube chat function you can be interactive too with the audience.